src/Kunstmaan/SeoBundle/Helper/Menu/SeoManagementMenuAdaptor.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Kunstmaan\SeoBundle\Helper\Menu;
+
+use Kunstmaan\AdminBundle\Helper\Menu\MenuItem;
+use Kunstmaan\AdminBundle\Helper\Menu\MenuBuilder;
+use Kunstmaan\AdminBundle\Helper\Menu\MenuAdaptorInterface;
+
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\Request;
+
+class SeoManagementMenuAdaptor implements MenuAdaptorInterface
+{
+    /** @var  Container */
+    private $container;
+
+    /**
+     * @param Container $container
+     */
+    public function __construct(Container $container)
+    {
+        $this->container        = $container;
```

Comment:
Why do you inject the container if you only use the security.context? Injecting the security.context should suffice.

You should never inject the full container unless it's really necessary (but most of the times it just isn't, and we should check other services as well)...